frontend/src/libs/components/app/app.tsx

```diff
@@ -43,7 +69,8 @@ const App: React.FC = () => {
 
   return (
     <>
-      <Header />
+      <Header menuItems={menuItems} isAuth={false} />
+
```

Comment:
I'm gonna rework auth accessor, and there will be a special hook to access the user from auth slice
const useAuthUser: () => AuthUserT | null = () => {
  return useAppSelector(selectUser);
};

so i guess isAuth is not required here, we will use useAuthUser to check authentication status inside header component

Reply:
Yes, I am sure that this flag will not be needed here, but I need it at the moment to somehow separate the sign in button and the burger menu. When your hook is in dev, I will remove this flag.

Reply:
Why not use `selectUser` which is already in dev?

Reply:
Yes, I remembered this later and made changes